src/views/registrations/View.svelte

```diff
@@ -124,7 +124,7 @@
           name: "id",
           label: "Radicle",
           validate: "identity",
-          placeholder: "Radicle ID, eg. rad:git:hnrkqdpm9ub19oc8d…",
+          placeholder: "Radicle ID, eg. rad:zKtT7DmF9H34KkvcKj…",
```

Comment:
this will show new format on the (old) app? :confused:

Reply:
a6de621a91ea760a934ecf4d48418fa91a07a786